### ISSUE TITLE
Person + comma lumping

### ISSUE THREAD

**Issue by soyjavi** [nlp-compromise/nlp_compromise#225]
I've upgraded to 6.2.1 and now lexicon doesn't work ... any idea?.

My previous code:

```
let lexicon = Compromise.lexicon();
lexicon['ava', 'AVA', 'Ava'] = 'Person';
```

**Comment by spencermountain** [comment]
hey Javi, here's the instructions about how to add words to the lexicon
https://github.com/nlp-compromise/nlp_compromise/wiki/Lexicon#adding-words
it's a hash,

**Comment by soyjavi** [comment]
Ey guy, don't close it,

I'm so sorry but you have a bug, this is my complete code:

```
import Compromise from 'nlp_compromise';

const sentence = 'Ava, Do you know if tomorrow will rain in Bangkok?';
let lexicon = Compromise.lexicon();
lexicon['ava', 'AVA', 'Ava'] = 'Person';

const tags = Compromise.sentence(sentence, {lexicon}).tags();
>  [ 'Noun',
  'Infinitive',
  'Person',
  'Infinitive',
  'Condition',
  'Date',
  'Modal',
  'Noun',
  'Preposition',
  'Place' ]
```

**Comment by spencermountain** [comment]
hi Javi, please look more carefully at the docs.
`lexicon['ava', 'AVA', 'Ava'] = 'Person'`
is invalid javascript!
`lexicon['ava']='Person'`

**Comment by 2426021684** [comment]
@spencermountain It is valid, it just uses the last item in this car "Ava". Just tested it in node repl.

**Comment by spencermountain** [comment]
yeah, the lexicon needs lowerCase
```
var Compromise =require('nlp_compromise')
const sentence = 'Ava, Do you know if tomorrow will rain in Bangkok?';
let lexicon = Compromise.lexicon();
lexicon['ava'] = 'Person';
const tags = Compromise.sentence(sentence, {lexicon}).tags();
// [ Person, ...
```

**Comment by soyjavi** [comment]
Advance but if you use `terms`:

```
Person {
  whitespace: { preceding: '', trailing: ' ' },
  text: 'Ava, Do you',
  normal: 'ava do you',
...
```

The text is `ava do you`???

**Comment by spencermountain** [comment]
fixed in v6.3.1 
(this was a pretty good one)
thanks

**Comment by soyjavi** [comment]
I love this library and I'm using it in hard-mode :) ... for now the only way I can help you is reporting issues :)
